### DISCUSSION TITLE
Option to update live wpm every keypress in addition to every second should be present.

### DISCUSSION THREAD

**Discussion by sanidhyas3s** [monkeytypegame/monkeytype#4431] [Ideas]
![image](https://github.com/monkeytypegame/monkeytype/assets/92792319/a93ab0c3-d3f1-4ca5-ba2d-cfe6ff17095a)

The per second one feels a bit laggy because the speed can change a lot every second, and since the update is not in sync with the typing it feels a bit weird. We can just also have an option to update it every character as an alternate to every second.

If this seems like something that should be done, I can happily work on implementing it myself.

**Comment by Miodec** [comment]
Updating the speed every keypress would be very hard to read, especially at higher speeds.

**Reply by sanidhyas3s** [reply]
Agree, but isn't that what we have with accuracy?

**Reply by Miodec** [reply]
Ok, I guess so. If you want to you can replace `true` with `per second` and `per keypress` for both live wpm and live acc.

**Reply by sanidhyas3s** [reply]
Ok, will do so and make a PR when I have it ready. Anything to keep in mind while I implement this?
